### ISSUE TITLE
Perhaps consider renaming it to cargo-hack?

### ISSUE THREAD

**Issue by asdf8dfafjk** [nakabonne/rhack#7]
Seeing as it deals with cargo more than rust and the usage of the term cargo is even encouraged for such things? This would hopefully let me just issue cargo hack and get done with things :). Small thing but I think it might matter

**Comment by nakabonne** [comment]
@asdf8dfafjk Hey, good point. I was looking into making this tool a sub-command of cargo, but I'm not familiar with it so I've been putting it off for a long time.

**Comment by asdf8dfafjk** [comment]
So yeah I just tried it and looks like renaming it cargo-hack alone won't be enough. So looks like our goal is same but the renaming step alone won't achieve it and you'll need to also modify the positional argument handling from the command line. 
Here's a sample command line session. I have verified that the error language is from your tool, thus the `cargo hack` part indeed calls `cargo-hack`. Ideally what is now `rhack edit <crate name>` would become `cargo hack <crate name>` so effectively you could ignore $2 and continue looking at $3. 

![image](https://user-images.githubusercontent.com/67588957/141484108-a1548e81-cea2-4649-8461-3d49e720b116.png)

**Comment by asdf8dfafjk** [comment]
I retract what I've said above. Give me a few minutes.

**Comment by asdf8dfafjk** [comment]
So this is the page for subcommands: https://doc.rust-lang.org/cargo/reference/external-tools.html?highlight=subcommand#custom-subcommands

Basically if you say `cargo hack chrono`, then $1 is `cargo-hack`, $2 is `hack` and $3 is `chrono`. If you say `cargo help hack`, then `$1 is `cargo-hack`, $2 is `hack`, $3 is `--help`. 

I'd propose the following alternatives for the current "sub-sub" commands:


```
rhack edit chrono -> cargo hack chrono
rhack --help             -> cargo help hack 
rhack help               -> cargo help hack
rhack undo chrono -> cargo hack chrono --undo

```

**Comment by simonsan** [comment]
`cargo-hack` is already existing, `cargo-rhack` would be free.

https://crates.io/crates/cargo-hack

**Comment by simonsan** [comment]
Fixed it, so it runs as a cargo subcommand and pushed it to https://crates.io/crates/cargo-rhack and invited @nakabonne as an owner (so they can update as they like, when they read this, took it just as a name reservation, so nobody comes now to take it). 👍🏽 

Fixed in #10 (closed due to branch rename)
Fixed in #11

**Comment by asdf8dfafjk** [comment]
Awesome.
